examples/Using_logprobs.ipynb

```diff
@@ -757,7 +760,63 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## 5. Conclusion"
+    "## 5. Calculating perplexity\n",
+    "\n",
+    "When looking to assess the model's confidence in a result, it can be useful to calculate perplexity, which is a measure of the uncertainty. Perplexity can be calculated by exponentiating the negative of the average of the logprobs. Generally, a higher perplexity indicates a more uncertain result, and a lower perplexity indicates a more confident result. As such, perplexity can be used to both assess the result of an individual model run and also to helpfully compare the relative confidence of results between model runs. While a high confidence doesn't guarantee result accuracy, it can be a helpful signal that can be paired with other evaluation metrics to build a better understanding of your prompt's behavior.\n",
```

Comment:
"is a measure of the model's uncertainty." 
.."average of the output logprobs" 
would remove "helpfully" before compare